### PR TITLE
Connect the kodflix app with our database! #28

### PR DESCRIPTION
**Pull request by walterperez**
Added MongoDB to the project.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "private": true,
   "dependencies": {
     "express": "^4.16.4",
+    "mongodb": "^3.2.3",
```
Comment (GawahChan) [inline package.json:7]: hello walter

File: src/backend/app.js [hunk 2]
```diff
@@ -1,24 +1,37 @@
...
-    res.send(movies);
-});
+  //Routes
+  app.get("/", (req, res) => {
+    res.sendFile(path.join(__dirname, "./../../build/", "index.html"));
+  });
+  app.get("/rest/shows", (req, res) => {
+    let collection = db.collection("shows");
+    collection.find({}).toArray(function(err, result) {
+      if (err) {
```
Comment (GawahChan) [inline src/backend/app.js]: Use ternary operator

**Review by GawahChan — CHANGES_REQUESTED** [review]
Hi Walter, just use ternary operator on app.js, line 23

File: src/backend/app.js (hunk 2 above)
Comment (walterperez) [reply src/backend/app.js]: Thanks Ill fix it now :)

**Review by GawahChan — APPROVED** [review]
Looks good, thanks Walter

**Pull request merged**